src/wp-includes/script-loader.php

```diff
@@ -34,6 +34,457 @@
+			continue;
+		}
+
+		$polyfill .= (
+			// Test presence of feature...
+			'( ' . $test . ' ) || ' .
+			// ...appending polyfill on any failures. Cautious viewers may balk
+			// at the `document.write`. Its caveat of synchronous mid-stream
+			// blocking write is exactly the behavior we need though.
+			'document.write( \'<script src="' .
+			esc_url( $scripts->registered[ $handle ]->src ) .
+			'"></scr\' + \'ipt>\' );'
+		);
+	}
+
+	return $polyfill;
+}
+
+function wp_default_packages_scripts( &$scripts, $suffix, $dev_suffix ) {
+	$packages_dependencies = array(
+		'api-fetch' => array( 'wp-polyfill', 'wp-hooks', 'wp-i18n' ),
+		'a11y' => array( 'wp-dom-ready', 'wp-polyfill' ),
+		'autop' => array( 'wp-polyfill' ),
+		'blob' => array( 'wp-polyfill' ),
+		'blocks' => array(
+			'wp-autop',
+			'wp-blob',
+			'wp-block-serialization-default-parser',
+			'wp-data',
+			'wp-deprecated',
+			'wp-dom',
+			'wp-element',
+			'wp-hooks',
+			'wp-i18n',
+			'wp-is-shallow-equal',
+			'wp-polyfill',
+			'wp-shortcode',
+			'lodash',
+			'wp-rich-text',
+		),
+		'block-library' => array(
+			'editor',
+			'lodash',
+			'moment',
+			'wp-api-fetch',
+			'wp-autop',
+			'wp-blob',
+			'wp-blocks',
+			'wp-components',
+			'wp-compose',
+			'wp-core-data',
+			'wp-data',
+			'wp-editor',
+			'wp-element',
+			'wp-html-entities',
+			'wp-i18n',
+			'wp-keycodes',
+			'wp-polyfill',
+			'wp-url',
+			'wp-viewport',
+			'wp-rich-text',
+		),
+		'block-serialization-default-parser' => array(),
+		'block-serialization-spec-parser' => array( 'wp-polyfill' ),
+		'components' => array(
+			'lodash',
+			'moment',
+			'wp-a11y',
+			'wp-api-fetch',
+			'wp-compose',
+			'wp-deprecated',
+			'wp-dom',
+			'wp-element',
+			'wp-hooks',
+			'wp-html-entities',
+			'wp-i18n',
+			'wp-is-shallow-equal',
+			'wp-keycodes',
+			'wp-polyfill',
+			'wp-url',
+			'wp-rich-text',
+		),
+		'compose' => array( 'lodash', 'wp-element', 'wp-is-shallow-equal', 'wp-polyfill' ),
+		'core-data' => array( 'wp-data', 'wp-api-fetch', 'wp-polyfill', 'wp-url', 'lodash' ),
+		'data' => array(
+			'lodash',
+			'wp-compose',
+			'wp-deprecated',
+			'wp-element',
+			'wp-is-shallow-equal',
+			'wp-polyfill',
+			'wp-redux-routine',
+		),
+		'date' => array( 'moment', 'wp-polyfill' ),
+		'deprecated' => array( 'wp-polyfill', 'wp-hooks' ),
+		'dom' => array( 'lodash', 'wp-polyfill', 'wp-tinymce' ),
+		'dom-ready' => array( 'wp-polyfill' ),
+		'edit-post' => array(
+			'jquery',
+			'lodash',
+			'postbox',
+			'media-models',
+			'media-views',
+			'wp-a11y',
+			'wp-api-fetch',
+			'wp-block-library',
+			'wp-blocks',
+			'wp-components',
+			'wp-compose',
+			'wp-core-data',
+			'wp-data',
+			'wp-deprecated',
+			'wp-dom-ready',
+			'wp-editor',
+			'wp-element',
+			'wp-embed',
+			'wp-i18n',
+			'wp-keycodes',
+			'wp-nux',
+			'wp-plugins',
+			'wp-polyfill',
+			'wp-url',
+			'wp-viewport',
+		),
+		'editor' => array(
+			'jquery',
+			'lodash',
+			// 'tinymce-latest-lists',
```

Comment:
@atimmer You said, this was already in Core, I guess we should register this as a script as well?

Reply:
We should, it's at `/wp-includes/js/tinymce/plugins/lists`.